playbooks/dependencies-fedora-restricted.yaml

```diff
@@ -45,6 +45,7 @@
       - udisks2
     state: present
     update_cache: "{{ true if zuul.attempts > 1 else false }}"
+    use_backend: "{{ 'dnf4' if zuul.attempts > 1 else 'auto' }}"
```

Comment:
Shouldn't this be `dnf5` ?

Reply:
I was trying to fall back to `dnf4` if the first attempt with `auto` failed for some reason, just like we force the DNF cache to be updated.  I tried to install `python3-libdnf5` to make Ansible's `dnf5` backend work, but I still ended up with:
```
TASK [Install RPM packages]
fedora-rawhide | MODULE FAILURE:
fedora-rawhide | Traceback (most recent call last):
fedora-rawhide |   File "<stdin>", line 107, in <module>
fedora-rawhide |   File "<stdin>", line 99, in _ansiballz_main
fedora-rawhide |   File "<stdin>", line 47, in invoke_module
fedora-rawhide |   File "<frozen runpy>", line 226, in run_module
fedora-rawhide |   File "<frozen runpy>", line 98, in _run_module_code
fedora-rawhide |   File "<frozen runpy>", line 88, in _run_code
fedora-rawhide |   File "/tmp/ansible_ansible.legacy.dnf5_payload_kecazv78/ansible_ansible.legacy.dnf5_payload.zip/ansible/modules/dnf5.py", line 708, in <module>
fedora-rawhide |   File "/tmp/ansible_ansible.legacy.dnf5_payload_kecazv78/ansible_ansible.legacy.dnf5_payload.zip/ansible/modules/dnf5.py", line 704, in main
fedora-rawhide |   File "/tmp/ansible_ansible.legacy.dnf5_payload_kecazv78/ansible_ansible.legacy.dnf5_payload.zip/ansible/modules/dnf5.py", line 487, in run
fedora-rawhide | AttributeError: 'Base' object has no attribute 'load_config_from_file'
```

After looking at [ansible.git](https://github.com/ansible/ansible) it made me think that its `dnf5` backend doesn't work with the current state of DNF5.